main/main.go

```diff
@@ -36,8 +36,8 @@ func main() {
 	go func() {
 		<-sigs
 		ctx.Log("event", fmt.Sprintf("Received shutdown request"))
-		killVMWatch(ctx, vmWatchCommand)
 		shutdown = true
+		killVMWatch(ctx, vmWatchCommand)
```

Comment:
this was here before your PR, but we should probably handle the error. 
```suggestion
		killVMWatch(ctx, vmWatchCommand)
```